source/mapping-rules/create-rules.txt

```diff
@@ -0,0 +1,105 @@
+====================
+Create Mapping Rules
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can create mapping rules to translate your relational tables into
+MongoDB collections. You can create mapping rules from either the 
+:guilabel:`Relational` or :guilabel:`MongoDB` headers of the schema model pane.
+
+Steps
+-----
+
+Complete the following steps to create a mapping rule.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Open a {+rel-mig+} project.
+
+   .. step:: Open the :guilabel:`Mappings` pane.
+
+      You can do this from the :guilabel:`Schema model` pane by clicking
+      on a collection under the :guilabel:`Relational` or :guilabel:`MongoDB`
+      headers.
+
+   .. step:: Click :guilabel:`+ Add` to create a new mapping rule.
+
+      If you opened this pane from the :guilabel:`Relational` header, 
+      this button is next to :guilabel:`Mappings to MongoDB Model`. If you
+      opened the pane from the :guilabel:`MongoDB` header, this button is
+      next to :guilabel:`Mappings from relational tables`.
+
+   .. step:: Define how you want to migrate the source table.
+
+      Options available on the :guilabel:`Add mapping` pane depend on the type of 
+      mapping rule you are adding. For a full overview of each mapping rule 
+      option see:
+
+      - :ref:`rm-new-documents`
+
+      - :ref:`rm-embedded-array`
+
+      - :ref:`rm-embedded-documents`
+
+   .. step:: Select the name of the source table or target collection.
+
+      If you opened this pane from the :guilabel:`Relational` header, input
+      the name of the target collection. If you opened the pane from 
+      the :guilabel:`MongoDB` header, select the name of the source table
+      from the menu.
```

Comment:
Oops sorry if my last comment made sure we should remove "drop-down." I think it'd still be helpful to mention it's a "drop-down menu" here since there's a lot of panels and menus floating around on the RM UI!